pkg/ui/react-app/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+import Navigation from './thanos/Navbar';
```

Comment:
shall we move this down to the local imports

Reply:
done